From ddd12ea150eef6524addfd4def1bf372b0d068e1 Mon Sep 17 00:00:00 2001
From: Zhang Xiao <xiao.zhang@windriver.com>
Date: Wed, 30 Nov 2011 09:28:38 +0800
Subject: [PATCH] fs: pramfs: Fixed double inode hashing in iget_locked path

Upstream ID c2062742f0cf80b94c9118b02fc19ef48effeb8f

git://pramfs.git.sourceforge.net/gitroot/pramfs/pramfs

Signed-off-by: Marco <marco.stornelli@gmail.com>
Integrated-by: Zhang Xiao <xiao.zhang@windriver.com>
---
 fs/pramfs/inode.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/fs/pramfs/inode.c b/fs/pramfs/inode.c
index 537a312..3f6125b 100644
--- a/fs/pramfs/inode.c
+++ b/fs/pramfs/inode.c
@@ -320,7 +320,6 @@ static int pram_read_inode(struct inode *inode, struct pram_inode *pi)
 	inode->i_mapping->a_ops = &pram_aops;
 	inode->i_mapping->backing_dev_info = &pram_backing_dev_info;
 
-	insert_inode_hash(inode);
 	switch (inode->i_mode & S_IFMT) {
 	case S_IFREG:
 		if (pram_use_xip(inode->i_sb)) {
-- 
1.7.0.4

